src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -65,20 +64,32 @@ export class SkyRadioGroupComponent implements AfterContentInit, ControlValueAcc
       }
     }
   }
-
   public get value(): any {
     return this._value;
   }
 
+  @Input()
+  public set tabindex(value: number) {
+    if (this._tabindex !== value) {
+      this._tabindex = value;
+      this.updateRadioButtonTabindexes();
+    }
+  }
+  public get tabindex(): number {
+    return this._tabindex;
+  }
+
   @ContentChildren(SkyRadioComponent, { descendants: true })
   private radios: QueryList<SkyRadioComponent>;
 
   private _name = `sky-radio-group-${nextUniqueId++}`;
   private _value: any;
+  private _tabindex: number;
 
   public ngAfterContentInit(): void {
     this.updateCheckedRadioFromValue();
     this.updateRadioButtonNames();
+    this.updateRadioButtonTabindexes();
```

Comment:
Uppercase `Indexes`:
`this.updateRadioButtonTabIndexes();`

Reply:
done